app/views/compute_resources/form/_gce.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= javascript 'foreman_google/json_loader' %>
```

Comment:
does it work for you? 
I tired also to use webpack by adding
`javascript *webpack_asset_paths('foreman_google', :extension => 'js')`